src/meson.build

```diff
@@ -36,6 +36,42 @@ configure_file(
     install_mode: 'rwxr-xr-x'
 )
 
+configure_file(
+    input: 'eos-kolibri-migrate.in',
+    output: 'endless-key-migrate',
+    configuration: endless_key_config,
+    install: true,
+    install_dir: bindir,
+    install_mode: 'rwxr-xr-x'
+)
+
+configure_file(
+    input: 'eos-kolibri-manage.in',
+    output: 'endless-key-manage',
+    configuration: endless_key_config,
+    install: true,
+    install_dir: bindir,
+    install_mode: 'rwxr-xr-x'
+)
```

Comment:
This is a duplicate of the above `configure_file`, right? Oh, looks like that was an existing mistake.

Reply:
Yes, duplicated simply.